content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Crie um Hostprocess do Windows
```

Comment:
Original title said "Windows HostProcess Pod".
Perhaps "Crie um Pod de HostProcess do Windows" delivers that better?